### ISSUE TITLE
Staging DSS is having trouble checking out certain files

### ISSUE THREAD

**Issue by kislyuk** [HumanCellAtlas/data-store#1871]
Example: https://dss.staging.data.humancellatlas.org/v1/files/10c29562-be77-4f42-b0aa-48017d9e5785?replica=aws&version=2018-09-17T161441.564206Z

**Comment by xbrianh** [comment]
Here is the contents of the example file:
```
<?xml version="1.0" encoding="UTF-8"?>
<Error><Code>NoSuchKey</Code><Message>The specified key does not exist.</Message><Key>blobs/77cf7757374ad67da42f9ae13dd3e3de14c341ad3a9af45294c7369b51b86114.b9f870487826de5207e2fa88ac747789d123a47d.61f6fb0dc7356587960fe3f57f88025d.9afd76a5</Key><RequestId>AEB6E06CC022B05B</RequestId><HostId>XobBYqTLOjBVsFtHySASGDwzAvshkUE7G/F84IXTiZJY6m5ZIi6HN70Qzwi1M2y4hrAqKOTkhmQ=</HostId></Error>
```

**Comment by kislyuk** [comment]
@xbrianh do you mean the contents of the response, or the body of the file?

The underlying file manifest does exist, at least:

```
>aws s3 cp s3://org-humancellatlas-dss-staging/files/10c29562-be77-4f42-b0aa-48017d9e5785.2018-09-17T161441.564206Z -
{"format": "0.0.4", "creator_uid": 8008, "version": "2018-09-17T161441.564206Z", "content-type": "application/json; dcp-type=\"metadata/process\"", "size": 391, "crc32c": "9afd76a5", "s3-etag": "61f6fb0dc7356587960fe3f57f88025d", "sha1": "b9f870487826de5207e2fa88ac747789d123a47d", "sha256": "77cf7757374ad67da42f9ae13dd3e3de14c341ad3a9af45294c7369b51b86114"}
```

**Comment by xbrianh** [comment]
@kislyuk The contents of the blob.

**Comment by kislyuk** [comment]
If that's the body of the file, then that looks like an uncaught error in PUT file, does it not?

**Comment by xbrianh** [comment]
@kislyuk I believe this is an artifact of #1636 

The question now is what to do about it?

**Comment by kislyuk** [comment]
Hold on, but the checkout was requested from the aws replica. I'm assuming the aws replica has the correct file contents and shouldn't be affected by the failure to sync to gcp?

**Comment by xbrianh** [comment]
The original file was likely clobbered during re-syncing efforts from a few months ago.

Bundles pointing to this file should be tombstoned, and wranglers requested to re-submit.

**Comment by xbrianh** [comment]
This will be resolved with #2167
